Presentation.Web/Tests/it-system/Catalog/Migration.System.e2e.spec.ts

```diff
@@ -0,0 +1,106 @@
+    });
+
+    it("Global Administrator is able to see the move button", () => {
+        loginHelper.loginAsGlobalAdmin();
+        checkMigrationButtonVisibility(true);
+    });
+
+    it("Local admin is not able to see the move button", () => {
+        loginHelper.loginAsLocalAdmin();
+        checkMigrationButtonVisibility(false);
+    });
+
+    it("Regular user is not able to see the move button", () => {
+        loginHelper.loginAsRegularUser();
+        checkMigrationButtonVisibility(false);
+    });
+
+    it("Global admin is able to get to the final migration window and execute a migration", () => {
+        loginHelper.loginAsGlobalAdmin();
+
+        var systemNameFrom = createItSystemName();
+        var systemNameTo = createItSystemName() + 1;
+        return pageObject.getPage()
+            .then(() => cataHelper.waitForKendoGrid())
+            .then(() => cataHelper.createCatalog(systemNameFrom))
+            .then(() => cataHelper.createCatalog(systemNameTo))
+            .then(() => toggleSystemActivation(systemNameFrom))
+            .then(() => openMigrationOnSpecificSystem(systemNameFrom))
+            .then(() => waitForModalWindowAnimation(constants.moveSystemButton))
+            .then(() => expect(element(cssHelper.byDataElementType(constants.moveSystemButton)).isPresent()).toBe(true))
+            .then(() => element(cssHelper.byDataElementType(constants.moveSystemButton)).click())
+            .then(() => waitForModalWindowAnimation(constants.consequenceButton))
+            .then(() => expect(element(cssHelper.byDataElementType(constants.consequenceButton)).isPresent()).toBe(true))
+            .then(() => select2SearchForSystem(systemNameTo))
+            .then(() => waitForSelect2DataAndSelect())
+            .then(() => element(cssHelper.byDataElementType(constants.consequenceButton)).click())
+            .then(() => waitForModalWindowAnimation(constants.startMigrationButton))
+            .then(() => expect(element(cssHelper.byDataElementType(constants.startMigrationButton)).isDisplayed()).toBe(true))
+            .then(() => element(cssHelper.byDataElementType(constants.startMigrationButton)).click())
+            .then(() => expect(element(cssHelper.byDataElementType(constants.startMigrationButton)).isDisplayed()).toBe(false));
+    });
+
+    function createItSystemName() {
+        return "It-System" + new Date().getTime();
+    }
+
+    function toggleSystemActivation(name: string) {
+        return pageObject.getPage()
+            .then(() => element(by.xpath('//*/tbody/*/td/a[text()="' + name + '"]/parent::*/parent::*//*/button')).click());
+    }
+
+    function openMigrationOnSpecificSystem(name: string) {
+        return pageObject.getPage()
+            .then(() => element(by.xpath('//*/tbody/*/td/a[text()="' + name + '"]/parent::*/parent::*//*/a[@data-element-type="usagesLinkText"]')).click());
+    }
+
+    function waitForModalWindowAnimation(name: string) {
```

Comment:
Den har et lidt skørt navn for den venter på alt muligt der ikke er begrænset af modal window. Den skal nok bare hedde det den gør